content/en/docs/instrumentation/net/exporters.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "Exporters"
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to export them to a backend.
+
+## Console exporter
+
+The console exporter is useful for development and debugging tasks, and is the simplest to set up.
+
+```
+dotnet add package OpenTelemetry.Exporter.Console
+```
+
+If you're using ASP.NET Core, configure the exporter in your ASP.NET Core services:
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOpenTelemetryTracing(b =>
+{
```

Comment:
This required additional nuget package be installed.

Reply:
Good catch, yeah, I can adjust this to have the `OpenTelemetry.Extensions.Hosting` package included above as a requirement